lab4/top5airlines.hql

```diff
@@ -6,6 +6,7 @@ USE HadoopLab4;
 
 !echo Creating tables...;
 
+-- GLC| The first line in `flights` the is the header, isn't?
```

Comment:
Header is actually filtered because it's not conformant to the scheme (i.e. first column should be int, etc.)
But I agree that if scheme changes in future, header could sneak into table, so I've updated repo - added setting to skip it in both tables